ui/docs/manual/tasks/groups.mdx

```diff
@@ -0,0 +1,26 @@
+---
+filename: tasks/groups.mdx
+title: Task Groups
+order: 50
+---
+
+# Task Groups
+
+Task Groups are a way to group tasks together based on their `taskGroupId`.
+When multiple tasks are created simultaneously, they often have the same `taskGroupId`.
+This is useful in situations where tasks are created for a Github
+push event with a [decision task](/docs/manual/design/conventions/decision-task).
```

Comment:
This has a good bit of overlap with https://docs.taskcluster.net/docs/manual/tasks/taskgroupid-schedulerid.  I like the idea of describing a taskgroup as a "thing" ("entity"? "resource"?) here.  In the existing docs, it's really just an id, and now that it can be sealed and there's a queue.getTaskGroup, that isn't really appropriate anymore.

Reply:
Yeah, I was not sure if adding sealing to that doc would make sense, so created a new page.
I will move information from that one and drop that (or maybe add redirect if it was linked somewhere)